components/ProductReviews.php

```diff
@@ -113,7 +113,7 @@ public function setData()
             ->when($this->property('variant'), function ($q) {
                 $q->where('variant_id', $this->property('variant'));
             })
-            ->when(Auth::getUser(), function ($q) {
+            ->when(Auth::getUser() && Auth::getUser()->customer, function ($q) {
```

Comment:
Let's use this line so we don't execute the `getUser` method twice:

```suggestion
            ->when(optional(Auth::getUser())->customer, function ($q) {
```